baseplate/server/wsgi.py

```diff
@@ -11,10 +11,10 @@
 
 try:
     # pylint: disable=no-name-in-module
-    from gevent.pywsgi import LoggingLogAdapter
+    from gevent.pywsgi import LoggingLogAdapter  # type: ignore
 except ImportError:
     # LoggingLogAdapter is from gevent 1.1+
-    class LoggingLogAdapter:
+    class LoggingLogAdapter:  # type: ignore
```

Comment:
Why are the type ignore flags needed here?

Reply:
Ah never mind. I see your comment above

Reply:
here's the upstream bug for more context https://github.com/python/mypy/issues/1153